manifests/install.pp

```diff
@@ -1,15 +1,15 @@
 # Install the foreman proxy
 class foreman_proxy::install {
   if ! $foreman_proxy::custom_repo {
-    foreman::repos { 'foreman_proxy':
+    class { '::foreman::repo':
```

Comment:
@domcleal you can also pass ```custom_repo``` to ```foreman::repo``` and let it handle it, but it will still manage EPEL and SCL then.

I feel like we need some sort of ```puppet-foreman_common``` to manage this. Have ```puppet-foreman``` and ```puppet-foreman_proxy``` simply include it. If a user wants to change the repo, it should be done on the common module.